src/components/Portrait/style.module.scss

```diff
@@ -0,0 +1,11 @@
+@use '../../themes/theme.scss';
+
+.portrait {
+  display: flex;
+  flex: 1;
+  box-shadow: theme.$red-box-shadow;
+  border-radius: 16px;
```

Comment:
this should also be a part of theme variables - so far I remember that repeats across the app